File: src/multiquines.java
import java.util.*;public class QC{public static void main(String[]A){String S="DATA";List L=Arrays.asList(S.split("\t"));int I=L.indexOf(A[0]);System.out.print(L.get(I+1)+S.replace("\\","\\\\").replace("\t","\\t").replace("\n","\\n").replace("\"","\\\"")+L.get(I+2));}}

File: src/multiquines.scala
object QC{def main(A:Array[String]){var S="DATA";var L=S.split("\t");var I=L.indexOf(A(0));print(L(I+1)+S.replaceAll("(\\\\|\\\")","\\\\$1").replace("\t","\\t").replace("\n","\\n")+L(I+2))}}
